grafana-plugin/src/pages/schedule/Schedule.tsx

```diff
@@ -190,44 +166,15 @@ class SchedulePage extends React.Component<SchedulePageProps, SchedulePageState>
                   {startMoment.format('DD MMM')} - {startMoment.add(6, 'day').format('DD MMM')}
                 </div>
               </HorizontalGroup>
-              {/*<HorizontalGroup width="auto">
```

Comment:
don't remove those my comments please, I'll uncomment and return them pretty soon